tests/TestDocs.hs

```diff
@@ -29,13 +30,15 @@ publishOpts = Publish.defaultPublishOptions
   where testVersion = ("v999.0.0", Version [999,0,0] [])
 
 main :: IO ()
-main = do
-  pushd "examples/docs" $ do
-    Docs.Package{..} <- Publish.unsafePreparePackage publishOpts
-    forM_ testCases $ \(P.moduleNameFromString -> mn, pragmas) ->
-      let mdl = takeJust ("module not found in docs: " ++ P.runModuleName mn)
-                         (find ((==) mn . Docs.modName) pkgModules)
-      in forM_ pragmas (flip runAssertionIO mdl)
+main = pushd "examples/docs" $ do
+  res <- Publish.preparePackage publishOpts
+  case res of
+    Left e -> preparePackageError e
```

Comment:
Yeah, I don't think it matters, as long as we choose one and stick to it. I think most of the compiler test suite uses stdout right now.

Maybe later we can get everything to use the framework that the psc-ide uses, then we wouldn't have to worry about issues like this. I'd be in favour of that, at least.